libraries/GCS_MAVLink/GCS.h

```diff
@@ -613,6 +614,84 @@ class GCS_MAVLINK
 
     uint8_t send_parameter_async_replies();
 
+#if HAVE_FILESYSTEM_SUPPORT
+
+    enum class FTP_OP : uint8_t {
+        None = 0,
+        TerminateSession = 1,
+        ResetSessions = 2,
+        ListDirectory = 3,
+        OpenFileRO = 4,
+        ReadFile = 5,
+        CreateFile = 6,
+        WriteFile = 7,
+        RemoveFile = 8,
+        CreateDirectory = 9,
+        RemoveDirectory = 10,
+        OpenFileWO = 11,
+        TruncateFile = 12,
+        Rename = 13,
+        CalcFileCRC32 = 14,
+        BurstReadFile = 15,
+        Ack = 128,
+        Nack = 129,
+    };
+
+    enum class FTP_ERROR : uint8_t {
+        None = 0,
+        Fail = 1,
+        FailErrno = 2,
+        InvalidDataSize = 3,
+        InvalidSession = 4,
+        NoSessionsAvailable = 5,
+        EOF_ = 6, // FIXME: Maybe don't send this in
```

Comment:
I think you need to add more words to

Reply:
I can't use `EOF` because it conflicts with AP_FS, and didn't really want anything longer. Any suggestions? (obviously the FIXME comment comes off of it)

Reply:
EndOfFile ?

Reply:
Sure.